desktop/apps/consignments2/stylesheets/landing/desktop.styl

```diff
@@ -65,6 +62,10 @@
     &__headline
       margin-bottom 38px
       garamond-size('xl-headline')
+
+      @media screen and (max-width: 768px)
```

Comment:
@craigspaeth - Is there a mixin for these responsive styles (asking because I'm about to start on some responsive work)